packages/plugin-prisma-utils/src/schema-builder.ts

```diff
@@ -93,6 +107,23 @@ schemaBuilder.prismaFilter = function prismaFilter<
   return ref as never;
 };
 
+schemaBuilder.prismaQueryModeEnum = function prismaQueryModeEnum() {
```

Comment:
can this be called something less generic, maybe `prismaStringFilterMode` or something, open to suggestions

Reply:
Sure, I adopted the enum name from the Prisma [query-engine](https://github.com/prisma/prisma-engines/blob/489dea78e5d64611960acfde15ac92fee3f21c41/query-engine/connectors/query-connector/src/filter/scalar/mod.rs#L30).

As the enum is defined in the `ScalarFilter` class there, maybe `prismaScalarFilterQueryModeEnum` would be a fitting name here.

Reply:
Naming is hard.  I think externally, I would want to avoid having Prisma in the typename (prisma is an implementation detail a consumer of the API shouldn't care about).    I'm a little surprised prisma uses such a generic name, but just `QueryMode` as the type name is likely too generic to use here, I think `StringFilterMode` (mode on `StringFilter`) might be good for the public name.  Maybe `prismaStringFilterModeEnum` as the method name

Reply:
I think scalar is probably not the right term here, since this only applies to strings